### ISSUE TITLE
Cloud preferences: accessibility issues

### ISSUE THREAD

**Issue by DavidBailes** [audacity/audacity#6208]
### Steps to reproduce

1. Open the Cloud page of preferences.
2. Press Alt so that access keys are shown.
3. Check that all controls have access keys.
4. With a screen reader running, tab to the person icon in the Account group box, and the "remove temporary files after" text box.

### Expected behavior

In step 3, all controls have access keys.
In step 4, the screen reader reads the label of the person icon, eg, account not linked, and accessibility name of the text box is "remove temporary files after days"

### Actual behavior

In step 3, not all controls have access keys.
In step 4, the label of the person icon is not read, and the word "days" is not included in the accessibility name of the text box.

### Audacity Version

current master / alpha / nightly build

### Operating system

Windows

**Comment by LWinterberg** [comment]
ah, we're in release again and the issue automation doesn't work anymore
